### PR TITLE
changes for plotting default args. 

### PR DESCRIPTION
**Pull request by aprotopopov**
Changes for `plot_history_alive` for python3. **BUG** fix for `expected_cum_transactions`

### CODE REVIEW

**Comment by aprotopopov** [comment]
Also added two plots for Heterogeneity in Dropout Probability and Heterogeneity in Transaction Rate. Mostly from BTYD [bgnbd.PlotDropoutRateHeterogeneity](https://rdrr.io/cran/BTYD/man/bgnbd.PlotDropoutRateHeterogeneity.html) and [bgnbd.PlotTransactionRateHeterogeneity](https://rdrr.io/cran/BTYD/man/bgnbd.PlotTransactionRateHeterogeneity.html). Added only for BG/NBD model but for Transaction Rate it could be also used for Pareto/NBD model.

**Comment by aprotopopov** [comment]
```
%matplotlib inline
from lifetimes import BetaGeoFitter
from lifetimes.datasets import load_cdnow_summary
from lifetimes.plotting import plot_transaction_rate_heterogeneity, plot_dropout_rate_heterogeneity
import pandas as pd
import matplotlib.pyplot as plt
plt.rcParams['figure.figsize'] = (15, 10)

cdnow = load_cdnow_summary()

bgf = BetaGeoFitter(penalizer_coef=0.0)
bgf.fit(cdnow['frequency'], cdnow['recency'], cdnow['T'])

plot_transaction_rate_heterogeneity(bgf)
plot_dropout_rate_heterogeneity(bgf)
```
image
image

File: lifetimes/utils.py [hunk 1]
```diff
@@ -209,7 +209,12 @@ def calculate_alive_path(model, transactions, datetime_col, t, freq='D'):
     customer_history = customer_history.set_index(datetime_col)
     # Add transactions column
     customer_history['transactions'] = 1
-    purchase_history = customer_history.resample(freq, how='sum').fillna(0)['transactions'].values
+    
+    # for some reason fillna(0) not working for resample in pandas with python 3.x,
```
Comment (CamDavidsonPilon) [inline lifetimes/utils.py]: would this change break python2?

Comment (aprotopopov) [inline lifetimes/utils.py]: No, tested with python 2 and 3. It's basically the same.

**Comment by CamDavidsonPilon** [comment]
👍 lgtm, thanks for this contribution!

**Pull request merged**